solr/packaging/build.gradle

```diff
@@ -109,6 +109,21 @@ task toDir(type: Sync) {
   }
 }
 
+task toTgz(type: Tar) {
```

Comment:
It is likely that this will require explicit file permissions to work properly on Windows machines. Don't know if tar task will accept a workaround like this one -

https://github.com/carrot2/carrot2/blob/master/distribution/build.gradle#L94-L98

Reply:
I just ran this in powershell on a fresh install of Solr. It worked fine for me. Is there some other way I should test it on windows to see the file permission issue?

Reply:
To be clear, I tested creating the tgz, not using it. I can try that out in a bit and see if the issues arise.

Reply:
Create a tgz in Windows and unpack it on Linux. Shell scripts should have the executable bit set.
This can be verified programmatically too, although the above is the ultimate check. :)

Reply:
Covered the scripts in `/bin`, the prometheus exporter and `server/scripts/cloud-scripts`. Not sure all of the locations I should check for.